gen/calc.py

```diff
@@ -670,15 +668,19 @@ def validate_exhibitor_storage_master_discovery(master_discovery, exhibitor_stor
                         # Use IAM Instance Profile for auth.
                         'rexray': {
                             'loglevel': 'info',
-                            'modules': {
-                                'default-admin': {
-                                    'host': 'tcp://127.0.0.1:61003'
-                                }
-                            },
-                            'storageDrivers': ['ec2'],
-                            'volume': {
-                                'unmount': {
-                                    'ignoreusedcount': True
+                            'service': 'ebs'
+                        },
+                        'libstorage': {
+                            'integration': {
+                                'volume': {
+                                    'operations': {
+                                        'mount': {
+                                            'preempt': True
```

Comment:
It's encouraging that the tests are passing, but I'm pretty sure we don't want to enable preemption for this test.